pkg/api/errors/state.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	grpcCodes "google.golang.org/grpc/codes"
+
+	kitErrors "github.com/dapr/kit/errors"
```

Comment:
We only use import aliases if there is a package import name conflict. We also don't use camel casing in package names- if we really needed an alias for `kit/errors`, we would use the alias `kiterrors`.

```suggestion
	"google.golang.org/grpc/codes"

	"github.com/dapr/kit/errors"
```